public/PlayGround/virtual-project/style.css

```
@@ -0,0 +1,1361 @@
+  /* visibility: hidden; */
+}
+
+.door {
+  width: 25%;
+  height: 75%;
+  background-image: url("Assets/photos/door.png");
+  background-position: bottom center;
+  bottom: 0;
+  right: 10%;
+  transform: translateZ(2vmin);
+  transition: all 0.3s ease;
```

Stylelint (17.5.0)

[error] Unexpected duplicate "transition" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)